src/app/app-routing.module.ts

```diff
@@ -82,7 +82,7 @@ const appRoutes: Routes = [
     },
     {
         path: 'object/:rid',
```

Comment:
@kilchenmann Could you tell me what `object` means? Does it refer to the selector? Do I need to change it to `resource-object`?